includes/class-webmention-receiver.php

```diff
@@ -554,6 +554,21 @@ public static function webmention_verify( $data ) {
 		$content_type  = wp_remote_retrieve_header( $response, 'Content-Type' );
 		$commentdata   = compact( 'remote_source', 'remote_source_original', 'content_type' );
 
+		// If this is an mf2 object store
+		if ( 'application/mf2+json' === $content_type ) {
+			$commentdata['remote_source_mf2'] = json_decode( $remote_source_original, true );
```

Comment:
will this be saved to the db? should we use an underscore as prefix, to make clear that it is a temp "attribute"?

Reply:
We don't for remote_source or remote_source_original, which are derived from pingbacks doing the same thing.

Reply:
It was not meant as a critic on this specific change. This was more a general question, but on a real world example. Why I asked: I think it is easier to understand/review the code when it is clear if the attribute is saved or not!?!

Reply:
I was just advising why I didn't. But at the moment, it isn't saved. I suppose we might if we ever tried for salmentions, but that would only need to save some basic comment data.